code/controllers/subsystem/abnormality_queue.dm

```diff
@@ -15,21 +20,26 @@ SUBSYSTEM_DEF(abnormality_queue)
 			possible_abnormalities[initial(abno.threat_level)] += abno
 	if(LAZYLEN(possible_abnormalities))
 		pick_abno()
-	addtimer(CALLBACK(src, .proc/HandleStartingAbnormalities), 120 SECONDS)
+	addtimer(CALLBACK(src, .proc/HandleStartingAbnormalities), 180 SECONDS)
 	..()
 
 /datum/controller/subsystem/abnormality_queue/fire()
 	if(world.time >= (wait*3))
-		if(!(HE_LEVEL in available_levels))
-			available_levels += HE_LEVEL
-	if(world.time >= (wait*6))
 		if(ZAYIN_LEVEL in available_levels)
 			available_levels -= ZAYIN_LEVEL
+		if(!(HE_LEVEL in available_levels) && world.time < (wait*9))
```

Comment:
Doesn't this mean that if it's BEFORE 45 minutes and HE arent in, they get added in?

Reply:
Yes.

Reply:
Neat

Reply:
Not sure if we have enough WAWs and Alephs for this, I'd keep HEs but make them much less common